SoftwareGuide/Latex/DevelopmentGuidelines/WriteAFilter.tex

```diff
@@ -521,9 +521,12 @@ \section{Filter Conventions}
 \end{minted}
 
 The default constructor should be \code{protected}, and provide sensible
-defaults (usually zero) for all parameters.  The copy constructor and
-assignment operator should be declared \code{private} and not implemented,
-to prevent instantiating the filter without the factory methods (above).
+defaults (usually zero) for all parameters. The copy constructor and
+assignment operator should not implemented in order to prevent instantiating
+the filter without the factory methods (above). They should be declared in the
+\code{public} section using the \code{ITK_DISALLOW_COPY_AND_ASSIGN} macro (see
```

Comment:
Escape underscores fro LaTeX.

Reply:
A sharp eye, again @thewtex ! Too strong. Thanks. To the 5.0 release we go ! 🎆